worker/codalabworker/local_run/local_run_manager.py

```diff
@@ -20,15 +20,26 @@ class LocalRunManager(BaseRunManager):
     container. It manages its cache of local Docker images and its own local
     Docker network.
     """
+
     # Network buffer size to use while proxying with netcat
     NETCAT_BUFFER_SIZE = 4096
     # Number of seconds to wait for bundle kills to propagate before forcing kill
     KILL_TIMEOUT = 100
     # Directory name to store running bundles in worker filesystem
     BUNDLES_DIR_NAME = 'runs'
 
-    def __init__(self, worker, docker, image_manager, dependency_manager,
-                 commit_file, cpuset, gpuset, work_dir, docker_network_prefix='codalab_worker_network'):
+    def __init__(
```

Comment:
Is there a way to prevent this from happening?

Reply:
This library has a strong tendency to start argument lists on new lines and separating each argument into its own line. I read that their rationale behind this style is to have minimal/clearer diffs when arguments change.
I'm personally neutral about this approach but if it's not something we want to adopt then we should probably reconsider using Black as it is really unavoidable and will probably happen anywhere many arguments are passed.

Reply:
I think that's logical.